src/lib/openapi/spec/project-dora-metrics-schema.ts

```diff
@@ -8,6 +8,11 @@ export const projectDoraMetricsSchema = {
     required: ['features'],
     description: 'A projects dora metrics',
     properties: {
+        projectAverage: {
+            type: 'number',
+            description:
```

Comment:
What are the units? Maybe we can include that in description. minutes, hours, days?

Reply:
Good suggestion